Add info to PREPARE statement docs (#17309)
pingcap/docs: sql-statements/sql-statement-prepare.md

|   UserVariable
```

> **Note:**
>

> > There limit for the number of placeholders is 65535

Reviewer: ```suggestion
> The maximum number of placeholders is 65535.
```

> The [`max_prepared_stmt_count`](/system-variables.md#max_prepared_stmt_count) system variable can be used to limit the number of `PREPARE` statements in the current TiDB instance.

Reviewer: ```suggestion
To limit the number of `PREPARE` statements in the current TiDB instance, you can use the [`max_prepared_stmt_count`](/system-variables.md#max_prepared_stmt_count) system variable.
```
